### PR TITLE
move toolchain image mirror in separate file

### PR DESCRIPTION
**Pull request by dipak-pawar**
*Changes Proposed in this PR*
- Move toolchain image mirror files in separate files
- Use toolchain secrets to push to quay.io/openshiftio

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -10,7 +10,7 @@ applyTemplate:
 	oc process -f $(WHAT) | oc apply -f -
 .PHONY: applyTemplate
 
-postsubmit-update: prow-services origin-release ci-infra-imagestreams libpod prow-monitoring build-dashboards-validation-image image-mirror-setup knative-image-mirror-setup
+postsubmit-update: prow-services origin-release ci-infra-imagestreams libpod prow-monitoring build-dashboards-validation-image image-mirror-setup knative-image-mirror-setup toolchain-image-mirror-setup
```
Comment (droslean) [inline Makefile]: Please remove it from here.

**Comment by droslean** [comment]
/hold

Hold until we store the secret to bitwarden.

File: Makefile (hunk 1 above)
Comment (petr-muller) [reply Makefile]: Why? Knative is doing the same thing, should not break anything.

Comment (droslean) [reply Makefile]: oh I didn't notice :).

**Pull request merged**